libantworld/renderer.cc

```diff
@@ -249,13 +249,13 @@ void Renderer::generateCubeFaceLookAtMatrices()
     }
 }
 //----------------------------------------------------------------------------
-void Renderer::applyFrame(float x, float y, float z,
-                          float yaw, float pitch, float roll)
+void Renderer::applyFrame(meter_t x, meter_t y, meter_t z,
+                          degree_t yaw, degree_t pitch, degree_t roll)
 {
-    glRotatef(roll, 0.0f, 1.0f, 0.0f);
-    glRotatef(pitch, 1.0f, 0.0f, 0.0f);
-    glRotatef(yaw, 0.0f, 0.0f, 1.0f);
-    glTranslatef(-x, -y, -z);
+    glRotated(roll.value(), 0.0, 1.0, 0.0);
```

Comment:
In this case it's really not important as it's only done once a frame, but as this probably ends up being done on the GPU, using double here is probably not a great plan.